ui/app/templates/components/generated-item-list.hbs

```diff
@@ -0,0 +1,81 @@
+<PageHeader as |p|>
+  <p.top>
+    {{#key-value-header path="vault.cluster.access.methods"}}
+      <li>
+        <span class="sep">
+          /
+        </span>
+        {{#link-to "vault.cluster.access.methods"}}
+          methods
+        {{/link-to}}
+      </li>
+    {{/key-value-header}}
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{capitalize method}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+{{section-tabs model "authShow" paths}}
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink @params={{array
+        "vault.cluster.access.method.item.create"
+        (pluralize itemType)
+      }}>
+      Create
+      {{itemType}}
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
+<ListView @items={{model}} @itemNoun={{itemType}} as |list|>
+  {{#if list.empty}}
+    <list.empty @title="No {{pluralize itemType}} yet"
+      @message="A list of {{pluralize itemType}} will be listed here. Create your first {{itemType}} to get started.">
+      {{#link-to
+        "vault.cluster.access.method.item.create"
+        (pluralize itemType)
+        tagName="button"
```

Comment:
Why the button here? Need to look and see if it’s a button elsewhere, but I think it should be an anchor.